--- gtk-engines-2.20.2/configure.ac.bero	2013-01-15 22:36:41.302006534 +0100
+++ gtk-engines-2.20.2/configure.ac	2013-01-15 22:36:53.562858650 +0100
@@ -166,7 +166,7 @@ fi
 
 AC_SUBST(DEVELOPMENT_CFLAGS)
 
-AM_CONFIG_HEADER([engines/support/config.h])
+AC_CONFIG_HEADERS([engines/support/config.h])
 
 AC_CONFIG_FILES([
 Makefile
--- gtk-engines-2.20.2/gtk-engines-cleanice-2.4.0/configure.ac.bero	2013-01-15 22:37:03.750735715 +0100
+++ gtk-engines-2.20.2/gtk-engines-cleanice-2.4.0/configure.ac	2013-01-15 22:37:12.476630543 +0100
@@ -5,8 +5,8 @@ AC_INIT(gtk-engines-cleanice, 2.4.0)
 AC_CONFIG_SRCDIR(config.h.in)
 AC_COPYRIGHT([Copyright 2002-2005 Elysium Project])
 
-AM_INIT_AUTOMAKE(AC_PACKAGE_NAME, AC_PACKAGE_VERSION)
-AM_CONFIG_HEADER(config.h)
+AM_INIT_AUTOMAKE
+AC_CONFIG_HEADERS([config.h])
 
 AM_MAINTAINER_MODE
 AC_DISABLE_STATIC
--- gtk-engines-2.20.2/gtk-flat-theme-2.0/configure.in.bero	2013-01-15 22:38:31.006701616 +0100
+++ gtk-engines-2.20.2/gtk-flat-theme-2.0/configure.in	2013-01-15 22:38:34.466659896 +0100
@@ -12,7 +12,6 @@ ACLOCAL="$ACLOCAL $ACLOCAL_FLAGS"
 AC_CANONICAL_HOST
 
 AC_PROG_CC
-AM_PROG_CC_STDC
 AC_PROG_INSTALL
 AC_PROG_MAKE_SET
 
